modules/otel-troubleshoot-metrics.adoc

```diff
@@ -6,14 +6,22 @@
 [id="exposing-metrics_{context}"]
 = Exposing the metrics
 
-The OpenTelemetry Collector exposes the metrics about the data volumes it has processed. The following metrics are for spans, although similar metrics are exposed for metrics and logs signals:
+The OpenTelemetry Collector exposes the following metrics about the data volumes it has processed:
```

Comment:
I'm not going to stop merge review over this, but I will note: with this many entries now, the description list is quite long and perhaps starting to become more burdensome to read than it is easy. Would you consider changing this description list to a proper table in another PR? 

The bright side is that if the follow up PR only has this formatting change, it would not need SME/QE review (or even peer review if you wait until next week 😉)